src/components/inscripcionTurnos/InscripcionActividad.js

```diff
@@ -92,19 +97,29 @@ export default function Actividad() {
 
   const guardarInscripcion = async () => {
     setIconoCargando(true);
-    await create({
-      actividadId: informacionSeleccionada.actividad.id,
-      medioDeTransporte: informacionSeleccionada.medioDeTransporte,
-      usuarioId: usuario.id,
-    });
-    notificarActualizacionTurno();
-    notificarActualizacionActividades();
-    history.push('/turnos/confirmacion');
+    try {
+      await create({
+        actividadId: informacionSeleccionada.actividad.id,
+        medioDeTransporte: informacionSeleccionada.medioDeTransporte,
+        usuarioId: usuario.id,
+      });
+      notificarActualizacionTurno();
+      notificarActualizacionActividades();
+      history.push('/turnos/confirmacion');
+    } catch (error) {
+      if (path(['response', 'status'], error) === 422) {
+        setErrorAlPedirTurno({
+          hayError: true,
+          mensaje: error.response.data.error,
+        });
+      }
```

Comment:
En caso que no sea un 422, deberías volver a tirar el error. Si no, quedaría atrapado acá sin ningún efecto en la vista.


```suggestion
      } else {
        throw error;
      }
```